Css test

Allenlixin opened the pull request:
web font and  typesetting a homepage
https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts

Allenlixin committed:
test css comprehension

Allenlixin committed:
web font

Allenlixin committed:
test homepage about community school https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Typesetting_a_homepage

Allenlixin committed:
styleing table https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_boxes/Styling_tables

junhuif reviewed (commented):
不错，好的开始，Merge了接着整。

下次的提升点：

1. 把这个Repo用 github pages 跑起来，让这些个页面能被访问到。可以方便看一下这些个页面是否能正确的在线上工作。
2. Commit message 里不要带 URL，URL 写在 commit description 里（也就是下一行）
3. 不要直接commit 在master分支，改动前都直接先  checkout 新的分支


这个PR能看出来是因为有Commit在Master分支后再切出来的，所以分支的Diff看起来就会比较乱。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+    font-size: 1.4rem;
+    line-height: 1.6;
+    word-spacing: 0.6rem;
+}
```
手写代码时注意空行。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+
```
顶部不空行。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+@font-face{
+    font-family: "my-font";
+    src:url("AlexBrush-Regular.ttf");
+    /* src:url("KaushanScript-Regular.otf"); */
```
删除无意义的代码。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+    src:url("AlexBrush-Regular.ttf");
+    /* src:url("KaushanScript-Regular.otf"); */
+}
+@font-face{
```
规则与规则之间加一个空行。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+}
+@font-face{
+    font-family: "header-font";
+    src:url("KaushanScript-Regular.otf");
```
属性与值之前有个空格 `src: url("...");`

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+
+p {
+   
+
```
不要无意义的空行。

junhuif commented on css/web-font/web-font-start.css:
```diff
@@ -0,0 +1,45 @@
+    font-size:2rem;
+}
+
+
```
多余的空行

junhuif commented on css/styling-table/style.css:
```diff
@@ -0,0 +1,86 @@
+    table-layout: fixed;
+    width:100%;
+    border-collapse: collapse;
+     border: 3px solid purple;
```
不对齐的缩进。

junhuif reviewed (commented):
Pretter 得配置一下保存时自动 Format. https://prettier.io/docs/en/index.html 它默认是支持HTML跟CSS的。

Allenlixin committed:
creat fancy letterheaded paper

https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_boxes/Creating_fancy_letterheaded_paper
